client/tokenized-payment-request/cart-api.js

```diff
@@ -0,0 +1,195 @@
+/* global jQuery */
+
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { applyFilters } from '@wordpress/hooks';
+
+export default class PaymentRequestCartApi {
+	// Used on product pages to interact with an anonymous cart.
+	// This anonymous cart is separate from the customer's cart, which might contain additional products.
+	// This functionality is also useful to calculate product/shipping pricing (and shipping needs)
+	// for compatibility scenarios with other plugins (like WC Bookings, Product Add-Ons, WC Deposits, etc.).
+	cartRequestHeaders = {};
+
+	/**
+	 * Creates an order from the cart object.
+	 * See https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/StoreApi/docs/checkout.md#process-order-and-payment
+	 *
+	 * @param {{
+	 *          billing_address: Object,
+	 *          shipping_address: Object,
+	 *          customer_note: string?,
+	 *          payment_method: string,
+	 *          payment_data: Array,
+	 *        }} paymentData Additional payment data to place the order.
+	 * @return {Promise} Result of the order creation request.
+	 */
+	async placeOrder( paymentData ) {
+		return await apiFetch( {
+			method: 'POST',
+			path: '/wc/store/v1/checkout',
+			credentials: 'omit',
+			headers: {
+				'X-WooPayments-Express-Payment-Request': true,
+				'X-WooPayments-Express-Payment-Request-Nonce':
+					window.wcpayPaymentRequestParams.nonce
+						.tokenized_cart_nonce || undefined,
```

Comment:
```suggestion
				'X-WooPayments-Express-Payment-Request-Nonce': getPaymentRequestData( 'nonce' )?.tokenized_cart_nonce,
```
Not really a big deal, but we could use our `frontend-utils.js` utility function here (down below inside `updateCustomer()` too). 🤷

Reply:
Sure, I updated all instances in b92ed0f75